onnxscript/version_converter/_version_converter.py

```diff
@@ -220,6 +220,38 @@ class _VersionConverter:
     def __init__(self, target_version: int):
         self.target_version = target_version
 
+    def _maybe_set_opset_version(
```

Comment:
A couple of questions:
* I assume that the version-converter will not change anything for opsets other than the standard onnx opset. Is this correct? If so, wouldn't it be better if it updates only the standard onnx opset version in imports?
* I don't understand the use of "max" down below. 
* I guess it might have to do with a failure to update some nodes to the requested opset version? It would be useful if we have some documentation about what happens in the various special cases, like failure to version-convert etc.

Reply:
For the first point, that is correct. I'm just returning, if node's domain is not the standard onnx opset.

For the second question, yes, the "max" is call to account for the failure of up-converts of some nodes. So for example, if GroupNorm fails to up-convert from 20->21, we wouldn't want opset imports to be "":21,

However I think this is a flaw with the current logic, we ideally don't want to have nodes with mixed opset versions, so the loop order might need to change (mentioned in the reply to Justin above)